products/gerrit.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Gerrit Code Review
+category: server-app
+iconSlug: "gerrit"
+permalink: /gerrit
+versionCommand: "java -jar gerrit.war version"
+releasePolicyLink: https://www.gerritcodereview.com/support.html
+changelogTemplate: https://www.gerritcodereview.com/__RELEASE_CYCLE__.html#{{"__LATEST__"| replace:'.',''}}
+activeSupportColumn: true
+releaseColumn: true
+releaseDateColumn: true
+
+identifiers:
+-   purl: pkg:docker/gerritcodereview/gerrit
+
+auto:
+-   git: https://github.com/GerritCodeReview/gerrit.git
+    regex: ^v(?<major>[0-9]+)\.(?<minor>[0-9]+)\.(?<patch>[0-9]+)(\.(?<tiny>[0-9]+))?$
+
+releases:
+-   releaseCycle: "3.7"
+    releaseDate: 2022-11-09
+    support: true
+    eol: false
+    latest: "3.7.0"
+    latestReleaseDate: 2022-11-09
+-   releaseCycle: "3.6"
+    releaseDate: 2022-05-23
+    support: true
+    eol: false
+    latest: "3.6.3"
+    latestReleaseDate: 2022-11-09
+-   releaseCycle: "3.5"
+    releaseDate: 2021-12-23
+    support: true
+    eol: false
+    latest: "3.5.4"
+    latestReleaseDate: 2022-11-08
+-   releaseCycle: "3.4"
+    releaseDate: 2021-5-17
+    support: 2022-11-09
+    eol: true
+    latest: "3.4.8"
+    latestReleaseDate: 2022-11-08
+-   releaseCycle: "3.3"
+    releaseDate: 2020-12-01
+    support: 2022-05-24
+    eol: true
+    latest: "3.3.9"
+    latestReleaseDate: 2021-12-23
+-   releaseCycle: "3.2"
+    releaseDate: 2020-06-01
+    support: 2021-12-07
+    eol: true
+    latest: "3.2.14"
+    latestReleaseDate: 2021-11-21
+-   releaseCycle: "3.1"
+    releaseDate: 2019-11-14
+    support: 2021-05-19
+    eol: true
+    latest: "3.1.16"
+    latestReleaseDate: 2021-08-18
+-   releaseCycle: "3.0"
+    releaseDate: 2019-05-15
+    support: 2020-12-01
+    eol: true
+    latest: "3.0.16"
+    latestReleaseDate: 2021-01-26
+-   releaseCycle: "2.16"
+    releaseDate: 2018-11-14
+    support: 2020-06-01
+    eol: true
+    latest: "2.16.28"
+    latestReleaseDate: 2021-11-19
+-   releaseCycle: "2.15"
+    releaseDate: 2018-03-28
+    support: 2019-11-15
+    eol: true
+    latest: "2.15.22"
+    latestReleaseDate: 2021-01-26
+-   releaseCycle: "2.14"
+    releaseDate: 2017-04-26
+    support: 2019-05-31
+    eol: true
+    latest: "2.14.22"
+    latestReleaseDate: 2020-11-18
+-   releaseCycle: "2.13"
+    releaseDate: 2016-09-21
+    support: false
+    eol: true
+    latest: "2.13.14"
+    latestReleaseDate: 2019-04-19
+-   releaseCycle: "pre 2.13"
+    releaseDate: 2015-12-22
+    support: false
+    eol: true
```

Comment:
Since this data is served on API as well, this just adds confusion. The general assumption is that any release not mentioned on the list is unsupported. So this isn't needed:

```suggestion
```

Reply:
done